soh/soh/resource/importer/scenecommand/SetMeshFactory.cpp

```diff
@@ -36,31 +36,39 @@ SetMeshFactory::ReadResource(std::shared_ptr<Ship::ResourceInitData> initData, s
     for (int32_t i = 0; i < polyNum; i++) {
         if (setMesh->meshHeader.base.type == 0) {
             PolygonDlist dlist;
+            dlist.opa = nullptr;
+            dlist.xlu = nullptr;
 
             int32_t polyType = reader->ReadInt8(); // Unused
             std::string meshOpa = reader->ReadString();
             std::string meshXlu = reader->ReadString();
 
-            auto opaRes = Ship::Context::GetInstance()->GetResourceManager()->LoadResourceProcess(meshOpa.c_str());
-            auto xluRes = Ship::Context::GetInstance()->GetResourceManager()->LoadResourceProcess(meshXlu.c_str());
-            dlist.opa = meshOpa != "" ? (Gfx*)(opaRes ? opaRes->GetRawPointer() : nullptr) : 0;
-            dlist.xlu = meshXlu != "" ? (Gfx*)(xluRes ? xluRes->GetRawPointer() : nullptr) : 0;
+            // Enables alt-toggling support by setting maintained c_str references to DList resource after pushing to vector the first
+            // Defers resource loading later in game when the scene is drawn
+            if (meshOpa != "") {
+                meshOpa = "__OTR__" + meshOpa;
```

Comment:
i feel like i remember there being a macro for this. it's easy enough to understand so i'm perfectly fine with these in this PR as-is, but if we're using a macro in other spots it'd be nice to do that here for consistency.

Reply:
Searching the repo, I couldn't find an example. The usage shown here at least aligns with all the existing usages that do this behavior in our importers (including an older one in this file on line 79)